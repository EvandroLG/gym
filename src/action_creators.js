import { SET_VIDEO_URL } from './actions';

export function setVideoUrl(videoUrl) {
  return {
    type: SET_VIDEO_URL,
    payload: videoUrl
  };
}
