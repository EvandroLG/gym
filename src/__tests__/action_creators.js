import { setVideoUrl } from '../action_creators';

describe('action creators', () => {
  it('should return an expected object', () => {
    const output = setVideoUrl('https://www.youtube.com/watch?v=04p4MWfhpAI');
    expect(output).toMatchSnapshot();
  });
});
