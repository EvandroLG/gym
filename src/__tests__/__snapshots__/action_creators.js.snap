// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`action creators should return an expected object 1`] = `
Object {
  "payload": "https://www.youtube.com/watch?v=04p4MWfhpAI",
  "type": "SET_VIDEO_URL",
}
`;
